web-bundle/pom.xml

```diff
@@ -64,10 +64,10 @@
             <artifactId>dropwizard-testing</artifactId>
             <scope>test</scope>
         </dependency>
-
         <dependency>
-            <groupId>com.wdtinc</groupId>
-            <artifactId>mapbox-vector-tile</artifactId>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java</artifactId>
+            <version>3.21.9</version>
```

Comment:
For some reason we need this to make the protobuf generated code in VectorTile.java compile (without it there is a compiler error due to the `parseUnknownField` method in VectorTile.java). Note that previously we excluded `protobuf-java` from com.wdtinc:mapbox-vector-tile explicitly. (?)